packages/rich-text/src/plugins/Hyperlink/EntryAssetTooltip.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import { css } from 'emotion';
+import tokens from '@contentful/forma-36-tokens';
+import {
+  EntityType,
+  BaseExtensionSDK,
+  ScheduledAction,
+} from '@contentful/field-editor-reference/dist/types';
+import { entityHelpers, Entry, Asset } from '@contentful/field-editor-shared';
+import { EntityStatusTag } from './EntityStatusTag';
+import { truncate } from '../../helpers/truncate';
+import { ScheduleTooltipContent } from './ScheduleTooltipContent';
+
+const styles = {
+  entityContentType: css({
+    color: tokens.colorTextLightest,
+    marginRight: tokens.spacingXs,
+    '&:after': {
+      content: '""',
+    },
+  }),
+  entityTitle: css({
+    marginRight: tokens.spacingXs,
+  }),
+  separator: css({
+    background: tokens.colorTextMid,
+    margin: tokens.spacingXs,
+  }),
+};
+
+interface EntryAssetTooltipProps {
+  id: string;
+  type: EntityType;
+  sdk: BaseExtensionSDK;
+}
+
+export function EntryAssetTooltip({ id, type, sdk }: EntryAssetTooltipProps) {
+  const [entityTitle, setEntityTitle] = React.useState('');
+  const [entityStatus, setEntityStatus] = React.useState('');
+  const [jobs, setJobs] = React.useState<ScheduledAction[]>([]);
+  const [isLoading, setIsLoading] = React.useState(true);
+  const [hasError, setHasError] = React.useState(false);
+
+  React.useEffect(() => {
+    async function loadContent() {
+      try {
+        setIsLoading(true);
+        setHasError(false);
+
+        const actions = {
+          Asset: {
+            getEntity: sdk.space.getAsset,
+            getEntityTitle: (args) => entityHelpers.getAssetTitle({ ...args, asset: args.entity }),
+          },
+          Entry: {
+            getEntity: sdk.space.getEntry,
+            getEntityTitle: (args) => entityHelpers.getEntryTitle({ ...args, entry: args.entity }),
+          },
+        };
+
+        let contentType;
+        const entity: Entry | Asset = await actions[type].getEntity(id);
+
+        if (entity.sys.contentType) {
+          const contentTypeId = entity.sys.contentType.sys.id;
+          const contentTypes = await sdk.space.getContentTypes();
```

Comment:
The original code uses `sdk.space.getContentType(id)` but I was getting an error from the sdk saying `getContentType` doesn't exist.